test/soak/cauldron/Makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY: cauldron
+
+TAG = 1.0
```

Comment:
To reduce the chance of an accidental push breaking stuff, can you look at the patterns in https://github.com/GoogleCloudPlatform/kubernetes/pull/9238 or at https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/kubectl-container/Makefile and use one of those patterns?